src/base/install/installDevTools.m

```diff
@@ -1,15 +1,17 @@
-function installDevTools()
+function installDevTools(repoName)
 % checks if the MATLAB.devTools are already present and installs the
 % MATLAB.devTools next to the cloned folder cobratoolbox (if not present yet)
 %
 % USAGE:
-%    installDevTools()
+%    installDevTools(repoName)
+%
+% INPUT:
+%    repoName:     Name of the repository (default: 'opencobra/cobratoolbox')
```

Comment:
If there is a default, then this is an optional input.

Reply:
True.